api/controllers/product.js

```diff
@@ -13,7 +13,6 @@ module.exports = {
         }
     },
     editProduct: async (req, res, next) => {
-        // const id = req.params.id;
         const { id, title, category, size, color, price, image } = req.body;
         try {
```

Comment:
add some validations

Reply:
I use the id from the body and it is validated, never to be empty. If there are more validations needed I'll add them.

Reply:
Okay cool! 😃